pkg/runtime/runtime_test.go

```diff
@@ -100,8 +124,115 @@ func TestNewInfo(t *testing.T) {
 						},
 					},
 				},
+				TemplateSpec: TemplateSpec{
+					PodSets: []PodSet{
+						{
+							Name:               constants.JobInitializer,
+							CountForNonTrainer: ptr.To[int32](1),
+							Containers:         make([]Container, 1),
+						},
+						{
+							Name: constants.JobTrainerNode,
+							Containers: []Container{{
+								Env: []corev1ac.EnvVarApplyConfiguration{{
+									Name:  ptr.To("TEST"),
+									Value: ptr.To("TEST"),
+								}},
+								Ports: []corev1ac.ContainerPortApplyConfiguration{{
+									Name:          ptr.To("http"),
+									Protocol:      ptr.To(corev1.ProtocolTCP),
+									ContainerPort: ptr.To[int32](8080),
+								}},
+								VolumeMounts: []corev1ac.VolumeMountApplyConfiguration{{
+									Name:      ptr.To("TEST"),
+									ReadOnly:  ptr.To(true),
+									MountPath: ptr.To("/var"),
+								}},
+							}},
+						},
+					},
+					ObjApply: jobsetv1alpha2ac.JobSetSpec().
+						WithReplicatedJobs(
+							jobsetv1alpha2ac.ReplicatedJob().
+								WithName(constants.JobInitializer).
+								WithTemplate(batchv1ac.JobTemplateSpec().
+									WithLabels(nil).
+									WithSpec(batchv1ac.JobSpec().
+										WithTemplate(corev1ac.PodTemplateSpec().
+											WithLabels(nil).
+											WithSpec(corev1ac.PodSpec().
+												WithContainers(
+													corev1ac.Container().
+														WithName(constants.ContainerDatasetInitializer).
+														WithVolumeMounts(
+															corev1ac.VolumeMount().
+																WithName(jobsetplgconsts.VolumeNameInitializer).
+																WithMountPath("/workspace/dataset"),
```

Comment:
We might need to store `/workspace/dataset` and `/workspace/model` in the constants since it is default paths to store dataset and model when our initializer is used.

Reply:
Sure, let me try it.

Reply:
Done.